zookeeper-kafka is not starting

Zookeeper kafka is not starting with the command `systemctl start zookeeper-kafka.service`.

The reason is the `jmx-exporter.yml` which is missing for this service.

```
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at sun.instrument.InstrumentationImpl.loadClassAndStartAgent(InstrumentationImpl.java:386)
	at sun.instrument.InstrumentationImpl.loadClassAndCallPremain(InstrumentationImpl.java:401)
Caused by: java.io.FileNotFoundException: /etc/zookeeper/zk-kafka-jmx-exporter.yml (No such file or directory)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at java.io.FileReader.<init>(FileReader.java:72)
	at io.prometheus.jmx.JmxCollector.<init>(JmxCollector.java:116)
	at io.prometheus.jmx.JavaAgent.premain(JavaAgent.java:55)
	... 6 more
```

--- a/tdp_vars_defaults/zookeeper-kafka/zookeeper-kafka.yml
+++ b/tdp_vars_defaults/zookeeper-kafka/zookeeper-kafka.yml
@@ -94,10 +94,9 @@ zk_start_on_boot: no
 zk_restart: "no"
 
 jmx_common_opts: "-Dcom.sun.management.jmxremote=true"
-zookeeper_jmx_exporter_conf_dir: /etc/zookeeper
-zookeeper_jmx_exporter_conf_file: zk-kafka-jmx-exporter.yml
+zookeeper_jmx_exporter_conf_file: jmx-exporter.yml
 jmx_exporter_zk_port: "{{ exporter_zookeeper_kafka_server_http_port }}"
-jmx_exporter_zk_opts: "{% if 'exporter_jmx' in groups and groups['exporter_jmx'] %}-javaagent:{{ jmx_exporter_install_file }}={{ jmx_exporter_zk_port }}:{{ zookeeper_jmx_exporter_conf_dir }}/{{ zookeeper_jmx_exporter_conf_file }}{% endif %}"
+jmx_exporter_zk_opts: "{% if 'exporter_jmx' in groups and groups['exporter_jmx'] %}-javaagent:{{ jmx_exporter_install_file }}={{ jmx_exporter_zk_port }}:{{ zookeeper_root_conf_dir }}/{{ zookeeper_jmx_exporter_conf_file }}{% endif %}"
 
 # jmx exporter configuration directory
 jmx_exporter_conf_dir: /etc/jmx-exporter

